### PR TITLE
Deploy improvements

### PR DESCRIPTION
**Pull request by sonicparke**
Trying to get Travis integrated.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "0.10"
+before_install: npm install -g grunt-cli
+install: npm install
+before_script: grunt build:hcsc
```
Comment (natemcdowel) [inline .travis.yml]: Don't need to run grunt here I think..

File: config/deploy.rb [hunk 2]
```diff
@@ -37,10 +37,11 @@
 set :copy_remote_dir, "#{shared_path}/tmp"
 
 set(:build_script){"""
+npm cache clean && \
```
Comment (mrchucho) [inline config/deploy.rb]: Are you sure this is necessary? I think we're effectively starting from scratch every time we do a build. Or, at least, we _should_ be.

File: .travis.yml (hunk 1 above)
Comment (sonicparke) [inline .travis.yml]: true but Travis should still be running

File: config/deploy.rb (hunk 2 above)
Comment (sonicparke) [inline config/deploy.rb]: no probably not neccessary

Comment (natemcdowel) [inline config/deploy.rb]: If we are cloning a new copy of the repo, I don't think it is necessary.

**Comment by natemcdowel** [comment]
:+1:

**Pull request merged**